docs/release-notes/self-hosted/self-hosted-v7.0.0.md

```diff
@@ -0,0 +1,122 @@
+---
+
+# Self-hosted v7.0.0
+
+These release notes are for [Codacy Self-hosted v7.0.0](https://github.com/codacy/chart/releases/tag/7.0.0){: target="_blank"}, released on April 4, 2022. <!-- TODO Update release date -->
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Upgrading Codacy Self-hosted
+
+Follow the steps below to upgrade to Codacy Self-hosted v7.0.0:
+
+1.  Check the [release notes for all Codacy Self-hosted versions](../index.md#self-hosted) **between your current version and the most recent version** for breaking changes and follow the instructions provided <span class="skip-vale">carefully</span>.
+
+1.  Follow the instructions to [upgrade your Codacy Self-hosted instance](https://docs.codacy.com/v7.0/chart/maintenance/upgrade/).
+
+1.  Update your Codacy command-line tools to the versions with the Git tag `self-hosted-7.0.0`:
+
+    -   [Codacy Analysis CLI](https://github.com/codacy/codacy-analysis-cli/releases/tag/self-hosted-7.0.0)
+    -   [Codacy Coverage Reporter](https://github.com/codacy/codacy-coverage-reporter/releases/tag/self-hosted-7.0.0)
+
+## Breaking changes
+
+**If you're using GitLab** please review the roles of your team on GitLab [considering the new permissions for project Maintainers](http://docs.codacy.com/v7.0/organizations/roles-and-permissions-for-synced-organizations/).
+
+GitLab defines Maintainers as [super-developers](https://about.gitlab.com/handbook/product/gitlab-the-product/#permissions-in-gitlab){: target="_blank"}:
+
+> They're able to push to <span class="skip-vale">master</span>, deploy to production. This role is often held by maintainers and engineering managers.
+
+However, until now the Codacy permissions for this role were limited to the same as a Developer. This change addresses customer feedback towards having a more seamless integration between Codacy and GitLab.
+
+!!! important
+    After upgrading Codacy, clean your browser cache and refresh the list of repositories on the **Repositories list** page to make sure that you're using the updated permissions:
+
+    ![Refreshing the list of repositories on Codacy](../images/cy-5876.png)
+
+## Product enhancements
+
+-   GitLab project members with the Maintainer role can now perform the following operations on Codacy:
+
+    -   Adding the project
+    -   Configuring the project
+    -   Changing the following analysis settings: ignoring issues and files, configuring code patterns, configuring file extensions, and managing branches
+
+    [See the updated permissions](https://docs.codacy.com/v7.0/organizations/roles-and-permissions-for-synced-organizations/) for all GitLab roles. (CY-5876)
+
+-   Included ESLint 8 as a new supported tool and deprecated ESLint 7. For now, all repositories will continue to use ESLint 7 by default. (CY-5848)
```

Comment:
Should we give a dead line for the deprecated tool to be removed? I bet many people will just continue using eslint 7 as that is what is configured, so should we say that next self hosted version or next self hosted breaking change we are removing it? Not sure how the timeline fits with the removal from cloud too.

Anyway I think it would be a good idea to say that we plan to remove it and to be clear that we expect users to change to the new eslint 8.

Reply:
It would be great to have it yes, but I don't think we know yet, but maybe mentioning something like "ESLint 7 support will be dropped on a future release" could reinforce that?

Reply:
@terciosoares, what should be our communication here for the Self-hosted customers?

Reply:
After discussing this with @terciosoares we agreed to provide a deadline for removing ESLint 7 from Codacy.

I updated the release note to state that:

* Codacy won't do more updates to ESLint 7
* Codacy will remove ESLint 7 from the product on April 4, 2023